src/topp/MzTabExporter.cpp

```diff
@@ -650,6 +650,23 @@ namespace OpenMS
 //        MzTabString db = sp.db.empty() ? MzTabString() : MzTabString(sp.db);
 //        MzTabString db_version = sp.db_version.empty() ? MzTabString() : MzTabString(sp.db_version);
 
+        // currently we don't save how many channels we saved in a consensus feature so we scan the file
+        // for the maximum max index in a consensus feature and take this as number of study variables
```

Comment:
mhh.. how about 
const FileDescriptions & 	getFileDescriptions () const
This should get you what you want, faster and also coverting files which have no featurehandle (unlikely to happen though... they are empty then I'd think).